go.mod

```diff
@@ -52,6 +52,7 @@ require (
 	github.com/hashicorp/mdns v1.0.4 // indirect
 	github.com/hashicorp/memberlist v0.2.3
 	github.com/hashicorp/net-rpc-msgpackrpc v0.0.0-20151116020338-a14192a58a69
+	github.com/hashicorp/protoc-gen-go-binary v0.0.0-20200316174223-e4c249d51dbb // indirect
```

Comment:
This is weird. Did you try `make update-vendor` to see if that indirectly strips this out (via `go mod tidy`)